src/app/pages/about-us/components/rating/rating.component.ts

```diff
@@ -0,0 +1,19 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-rating',
+  templateUrl: './rating.component.html',
+  styleUrls: ['./rating.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RatingComponent {
+  @Output() currentRateEvent = new EventEmitter<number>();
+  currentRate: number;
+  selectedStarIndex: number;
+
+  onStarRate(i: number):void {
```

Comment:
1. do not use single-letter variables/parameter names, write "index" here for example
2. add space after semicolon